lib/version_info.rb

```diff
@@ -0,0 +1,41 @@
+class VersionInfo
```

Comment:
This class is basically a semver parser right? Does it make sense to name it accordingly? What about: `class SemVerParser`?